pkg/controllers/etcd.go

```diff
@@ -55,8 +56,10 @@ func (s *EtcdService) Name() string           { return "etcd" }
 func (s *EtcdService) Dependencies() []string { return []string{} }
 
 func (s *EtcdService) configure(cfg *config.MicroshiftConfig) {
-	caCertFile := filepath.Join(cfg.DataDir, "certs", "ca-bundle", "ca-bundle.crt")
-	certDir := filepath.Join(cfg.DataDir, "certs", s.Name())
+	certsDir := filepath.Join(cfg.DataDir)
+
+	caCertFile := cryptomaterial.UltimateTrustBundlePath(certsDir)
+	etcdCertDir := filepath.Join(certsDir, s.Name())
```

Comment:
Tests are failing because etcd cannot start properly. It searches for 

`/var/lib/microshift/etcd/etcd-peer.crt`

when it should be:

`/var/lib/microshift/certs/etcd/etcd-peer.crt`

Reply:
Ooops, I forgot to replace the original `filepath.Join()`. That should now be fixed.